index.html

```diff
@@ -163,152 +163,8 @@ <h1>Community</h1>
     <p>There is a bunch of niche communities in the digital, design, coding spaces. Here is our list:</p>
 
```

Comment:
I know this is super minor but could we please remove the empty lines here for consistency.